Initial versions of sra and salmon calls (#1)
hoganlab-dartmouth/pa-seq-compendia: README.md

# pa-seq-compendia

> To build compendia of publicly available RNAseq data for P. aeruginosa, sepcifically.

Reviewer: ```suggestion
To build compendia of publicly available RNAseq data for *P. aeruginosa*, sepcifically.
```

> This short pipeline (collection of bash scripts) is written to download P. aeruginosa

Reviewer: ```suggestion
This short pipeline (collection of bash scripts) is written to download *P. aeruginosa*
```

RNAseq data from the NBCI SRA database and process them with salmon. It works off-of

> a pre-downloaded SRA run table and pre-determined salmon parameters.

Reviewer: Might be helpful to link to Jake's repository for reference. You can do this by 
```
text text [hyperlink name](URL) more text
```

Author: Good idea, thanks! I'll add a sentence to help relate it to Jake's as well to help with the big picture.

> Before use PBS script headers will need to be altered to run through a specific user

Reviewer: ```suggestion
Before using the PBS script, the headers will need to be altered to run through a specific user
```

account on Dartmouth's disocvery cluster and/or dirs in bash scripts will need to be

> restructured. 

Reviewer: Might want to link to the specific files where this needs to be changed

Author: Thanks - I hadn't thought to link anything, but it seems super useful, glad you suggested!

## Installation

If these scripts are run locally they require the SRAtoolkit and salmon.

All software necesary has been installed on dscovery by admin.

## Usage

> 1. Upload a run table from sra to the working dir.

Reviewer: I'm a little confused by this step and what this means. So are you copying data from the `sra` directory to your working dir?

Author: I had copied the run table from my local machine onto my discovery home dir. The run table itself, I downloaded from the SRA website doing a manual search and export. It is a step where I just did the fastest thing even though it doesn't make it super easy for someone to use..but I'll add more to this step of the docs so it is clear. Thanks!

eg. scp

2. Assemble a directory structure.

(to do)

> 3. Create a P. aeruginosa transcriptome index.

Reviewer: ```suggestion
3. Create a *P. aeruginosa* transcriptome index.
```
